### PR TITLE
[fix] coin lb italics #358

### LINKED ISSUE #358: Coin leaderboard shows names italicized
## Description of bug / unexpected behavior

As above.

## Expected behavior

This shouldn't happen - all the names should be regular text.

## How to reproduce the issue
`/coin leaderboard` (on the main server) produces
image

## Additional comments

This is most likely caused by some usernames in the leaderboard having underscores, which Discord interprets as italic text.

### PR DESCRIPTION
**Pull request by marko-polo-cheno**
## Summary of Changes
Enclose user names with backticks.

## Motivation and Explanation
Fixes lb italics due to special characters

## Related Issues
Resolves #358 

## Steps to Reproduce
.lb

## Demonstration of Changes
one line change

### CODE REVIEW

**Review by mcpenguin — COMMENTED** [review]
IMO, I'm not a fan of the "code blocks" that the usernames are now wrapped in; I think the previous implementation (with the regular text) works better.

Perhaps instead we can just replace all `_` with `\_` in the username?

**Comment by marko-polo-cheno** [comment]
> IMO, I'm not a fan of the "code blocks" that the usernames are now wrapped in; I think the previous implementation (with the regular text) works better.
> 
> 
> 
> Perhaps instead we can just replace all `_` with `\_` in the username?

True, I'll add a replace for _ * ~ and `

**Comment by marko-polo-cheno** [comment]
image

File: src/commandDetails/coin/leaderboard.ts [hunk 1]
```diff
@@ -38,7 +38,16 @@ const getCurrentCoinLeaderboardEmbed = async (
     }
     if (user.bot) continue;
     const userTag = user?.tag ?? '<unknown>';
-    const userCoinEntryText = `${leaderboardArray.length + 1}. ${userTag} - ${
+    const cleanUserTag = userTag
+      .split('~')
+      .join('\\~')
+      .split('*')
+      .join('*')
```
Comment (Picowchew) [inline src/commandDetails/coin/leaderboard.ts:45]: This should be `\\*` instead?

**Review by Picowchew — CHANGES_REQUESTED** [review]
I just left 1 comment.

File: src/commandDetails/coin/leaderboard.ts (hunk 1 above)
Comment (marko-polo-cheno) [reply src/commandDetails/coin/leaderboard.ts:45]: darn i wanted to call yall out for skim review 💀

**Review by Picowchew — APPROVED** [review]
LGTM!

**Comment by mcpenguin** [comment]
Looks good to me! In the future, please do `git commit --amend` and `git push -f` to make the commit history nicer :D

**Review by mcpenguin — APPROVED** [review]

**Comment by marko-polo-cheno** [comment]
i havent had the honour of clicking merge in... a phat minute

**Pull request merged**